app/api/groups/get_group.go

```diff
@@ -98,26 +98,29 @@ func (srv *Service) getGroup(w http.ResponseWriter, r *http.Request) service.API
 
 	query := srv.Store.Groups().
 		Joins(`
-			LEFT JOIN groups_ancestors_active
-				ON groups_ancestors_active.child_group_id = groups.id AND
-					groups_ancestors_active.ancestor_group_id = ?`, user.OwnedGroupID).
+			LEFT JOIN ? AS manager_access ON 1`,
+			srv.Store.GroupAncestors().ManagedByUser(user).
+				Where("groups_ancestors.child_group_id = ?", groupID).
+				Select("1 AS found").Limit(1).SubQuery()).
```

Comment:
Still the same remark applies. So:
```sql
		  LEFT JOIN ? AS manager_access ON child_group_id = groups.id`,
			srv.Store.GroupAncestors().ManagedByUser(user).Select("1 AS found, groups_ancestors.child_group_id").SubQuery()).
```

Reply:
Is there a reason for changing the query this way besides the joke about DB professors killing?

Reply:
Because it is not sane SQL to write a regular left join with the filter inside and "1" as "ON" condition. 
Anyway, having the limit in a subquery is quite of an anti-pattern, especially when it can be avoided. (here it has probably no impact)

Reply:
I think it is sane, but never mind. Fixed